sparkflow/HogwildSparkModel.py

```diff
@@ -84,9 +84,10 @@ def handle_model(data, graph_json, tfInput, tfLabel=None,
                 put_deltas_to_server(gradients, master_url)
 
             if verbose:
+                partition_id = uuid.uuid4().hex
```

Comment:
Do you want this id to be the same for each iteration within a partition?

Reply:
yeah, whoops